src/actions/getMintlist.ts

```diff
@@ -14,21 +16,36 @@ const defaultIpfsGateway = "https://ipfs.io/ipfs/";
 
 export const getMintlist = async (config: Config) => {
   let mintlist: Maybe<Mintlist>;
+  let merkleTreeIndex = 0;
+
+  const airWild2Sale = await getAirWild2SaleContract(
+    new ethers.VoidSigner(""),
+    config.contractAddress
+  ); // Since we'll only be reading here we actually don't need a signer
+
+  if (airWild2Sale) {
+    merkleTreeIndex = (await airWild2Sale.currentMerkleRootIndex()).toNumber();
+  }
 
   // fetch via main uri
   try {
-    const res = await fetch(config.merkleTreeFileUri, { method: "GET" });
+    const res = await fetch(config.merkleTreeFileUris[merkleTreeIndex], {
+      method: "GET",
+    });
     mintlist = (await res.json()) as Mintlist;
     return mintlist;
   } catch (e) {
     console.error(
-      `Unable to fetch mint list via uri ${config.merkleTreeFileUri}`
+      `Unable to fetch mint list via uri ${config.merkleTreeFileUris}`
```

Comment:
You're printing out an array of strings into the console. This wouldn't be a very helpful error message since we have no idea which uri you were trying to access.